near/tests/stake_nodes.rs

```diff
@@ -17,7 +17,8 @@ fn test_stake_nodes() {
     init_test_logger();
 
     let mut genesis_config = GenesisConfig::testing_spec(2, 1);
-    genesis_config.epoch_length = 10;
+    //    genesis_config.block_producers_per_shard = vec![2];
```

Comment:
is this needed? to test 2 block producers?

Reply:
Oh just forgot to remove